app/_locales/en/messages.json

```diff
@@ -938,6 +938,12 @@
     "message": "Deposit $1",
     "description": "$1 represents the crypto symbol to be purchased"
   },
+  "deprecatedTestNetworksLink": {
+    "message": "Learn more"
+  },
+  "deprecatedTestNetworksMsg": {
+    "message": "Due to the protocol changes of Ethereum: Rinkeby, Ropsten, Kovan test networks may not work as reliably and will be deprecated soon."
```

Comment:
Nit but can we change this to `and Kovan`?

Reply:
Hi @darkwing, Done [here](https://github.com/MetaMask/metamask-extension/pull/15725/commits/a5bfc34927a75f7ea24ad05ac388835b0cc25115#diff-7509d7529ea1cbc57ef79713676f7d124d9d5a446053f94836876c4cd3f3ccd6).